### DISCUSSION TITLE
Limit on song download

### DISCUSSION THREAD

**Discussion by Hazhi-Hassan** [z-huang/music#1640] [Q&A]
Dear community, 

I am writing for help, since I was not able to find any solution. Since this new version, my library will only allow me to add up to 44 songs to it. If I try to add another one, it gives me "unknown error" message. I have removed a song and everything is good but when i have 44 I cannot even play any music. Can someone please help me with this?

Android 14, Nothing Phone 2a,

**Comment by 100Create** [comment]
Wrong place to post, put it in issues
